lldb/test/API/tools/lldb-dap/attach/TestDAP_attachByPortNum.py

```diff
@@ -18,6 +18,7 @@
 import socket
 
 
+@skip("SBTarget::ConnectRemote is not synchronous")
```

Comment:
Should we put a link to the bug to fix this test?

Reply:
I filed #138803